cartoframes/contrib/vector.py

```diff
@@ -8,14 +8,14 @@
 except ImportError:
     HAS_GEOPANDAS = False
 
-
 class QueryLayer:
     def __init__(self, query, color=None, size=None, time=None):
         self.query = query
         self.color = color
         self.size = size
         self.time = time
-
+        self.orig_query = query
+        self.is_basemap = False
```

Comment:
Same here -- is it needed or planned on being needed?

Reply:
Same story: for `_get_bounds()` function (found in context.py)

https://github.com/CartoDB/cartoframes/blob/2c5dec5c9bfe54aae78b4ee63b140ccb0588500b/cartoframes/context.py#L1772-L1774